build.js

```diff
@@ -1839,8 +1854,14 @@ var dirtyDetectingBuilder = function(rootPackageName, resultsCache, prevResultsC
         autogenAliases.genSourceFiles.internalInterface,
         autogenAliases.genSourceFiles.internalImplementation,
       ])
+      .concat(commonML.linkFlags)
```

Comment:
I added a function to merge defaults in, so we don't have to do those checks everywhere :)